Epsilon/Components/KpiTableComponent.cs

```diff
@@ -0,0 +1,149 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+using Epsilon.Abstractions;
+using Epsilon.Abstractions.Components;
+using System.Globalization;
+
+namespace Epsilon.Components;
+
+public class KpiTableComponent : AbstractCompetenceComponent
+{
+    public override async Task<Body?> AddToWordDocument(MainDocumentPart mainDocumentPart)
+    {
+        var body = mainDocumentPart.Document.Body;
+        
+        body.AppendChild(new Paragraph(
```

Comment:
Have the same check for null of body as in ComptenceProfileComponent or use question marks. One of the two